CONTRIBUTING.md

```
@@ -170,6 +170,24 @@ pnpm dev
 pnpm build
 ```
 
+#### DOCUMENTATIONS
+
+
+###  Backend Documentation
+ run
+ ```bash
+ pnpm dev
+ ```
+ The swaggerUI will be available on https://localhost:{PORT}/docs
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)